### PR TITLE
Updated TypeScript Comment

### PR DESCRIPTION
**Pull request by GuyGazit**

### CODE REVIEW

**Comment by lirook** [comment]
@GuyGazit what do mean by "recommended"? Does the user has to have the source files on disk?

**Comment by GuyGazit** [comment]
> @GuyGazit what do mean by "recommended"? Does the user has to have the source files on disk?

If the user doesn't have the sources on disk or `"inlineSources": true` it would create the same issue Otonomo had with finding alternative source files.

Technically we don't need them to have `"inlineSources": true`, but it is preferable just in case (like how the issue with Otonomo wasn't recreated when it was true).

File: docs/node-setup.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ If your application's code is being transpiled or bundled, you must include the
 
 - [**Webpack**](https://webpack.js.org/) - use either the `inline-source-map` or `source-map` values for the `devtool` option in the webpack config file ([reference](https://webpack.js.org/configuration/devtool)).
 - [**Babel**](https://babeljs.io/) - use either the `"inline"`, `"both"` or `true` values for the `sourceMaps` option in the Babel config file ([reference](https://babeljs.io/docs/en/options#sourcemaps)).
-- [**TypeScript**](https://www.typescriptlang.org/) - add either `"inlineSourceMap": true` or `"sourceMap": true` as well as `"inlineSources": true` in the TypeScript config file ([reference](https://www.typescriptlang.org/tsconfig#inlineSourceMap)).
+- [**TypeScript**](https://www.typescriptlang.org/) - adding either `"inlineSourceMap": true` or `"sourceMap": true` inside the TypeScript config file is a must. Using `"inlineSources": true` is recommended. ([reference](https://www.typescriptlang.org/tsconfig#inlineSourceMap)).
```
Comment (lirook) [inline docs/node-setup.md:96]:
```suggestion
- [**TypeScript**](https://www.typescriptlang.org/) - add either `"inlineSourceMap": true` or `"sourceMap": true` as well as `"inlineSources": true` in the TypeScript config file ([reference](https://www.typescriptlang.org/tsconfig#inlineSourceMap)). Note that `"inlineSources": true` can be left out if the source files are present along with the transpiled files on the machine your code runs on.
```

**Comment by lirook** [comment]
@GuyGazit after some more thought, we decided it's best not to document the files-on-disk option, and just ask to add "inlineSources" so I will close this for now.

**Pull request closed without merging**